helm-system-packages.el

```diff
@@ -373,7 +373,7 @@ Return the result as a string."
 
 With prefix argument, insert the output at point.
 Otherwise display in `helm-system-packages-buffer'."
-  (let ((res (helm-system-packages-call command args)))
+  (let ((res (apply 'helm-system-packages-run (cons command args))))
```

Comment:
`-print` is deprecated.  We should eventually move to `-show-information`.  But that can wait for the next pull request :)

Reply:
Yes, this is just lack of (immediate) effort on my part.